conanfile.py

```diff
@@ -2,6 +2,7 @@
 from conan.errors import ConanInvalidConfiguration
 from conan.tools.apple import is_apple_os
 from conan.tools.cmake import CMakeDeps, CMakeToolchain
+from conan.tools.build import cross_building
```

Comment:
please move one line above to maintain alphabetical order

Reply:
done